websearch/__main__.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from http.client import error
+from flask import Flask, redirect
+from gevent.pywsgi import WSGIServer
+from .script import WebSearch
+
+import sys
+
+
+APP = Flask(__name__)
```

Comment:
can you change variable APP to SERVER or server to more explicit ;)

Reply:
it's done